charts/castai-cluster-controller/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: castai-cluster-controller
 description: CAST AI cluster controller deployment chart.
 type: application
-version: 0.49.1
-appVersion: "v0.34.2"
+version: 0.49.2
+appVersion: "v0.34.3"
```

Comment:
Was it a bugfix or someething new added?
If it's not a bugfix `minor` version should be changes.
As per https://semver.org/

Reply:
Guess the image is released and here's not the place to talk about it. 
But please read thorugh https://semver.org/ for future reference.